<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCropWindow</class>
 <widget class="QWidget" name="ImageCropWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="line_imgAddr"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_imgFileDialog">
       <property name="text">
        <string>Відарыс</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_loadImage">
       <property name="text">
        <string>Запамп.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Геямэтрыя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_form">
       <item>
        <property name="text">
         <string>Круг</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Квадрат</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Трохкутнік</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Радыюс</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_radius">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Таўшчыня</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_crop_width">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Таўшчыня абрэзчыка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_resize_width"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_Crop">
       <property name="text">
        <string>Адсячы</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
